test/unit_tests/service/ClientSpec.coffee

```diff
@@ -52,6 +52,7 @@ describe 'z.service.Client', ->
         expect(client.status).toHaveBeenCalledTimes 1
         done()
       .catch done.fail
+      jasmine.clock().tick 10
```

Comment:
`10` is a pretty short value. Does it need to be that short or is `100` also ok?